src/assets/MobileTestAutomation/automationFrameworksComparison.md

```diff
@@ -0,0 +1,161 @@
+
+### When to use?
+
+- When developing iOS applications with stable, reliable, and fast testing
+- When your team is focused on iOS development
+- When you need to test iOS apps across various devices and screen sizes
+
+## [**Calabash**](https://github.com/calabash)
+
+Calabash is an open source framework that allows you to write and execute tests for Android and iOS apps. [Here](https://github.com/calabash) is the GitHub link. It uses
+Cucumber, a tool that lets you write tests in plain language.
+
+### Pros and Cons:
+
+| **Pros 👍**                                                                               | **Cons 👎**                                                                                                                                                                                   |
+|-------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Environment:** Can be installed and run on various OS such as Windows, Linux, and macOS | **Lack of documentation:** Its documentation is quite minimal, making it challenging to set up the automation framework. To complete the setup, one often needs to rely on external resources |
+| **Cross-Platform:** It supports writing tests for both iOS and Android                    | **Limited Language Support:** It supports only Ruby                                                                                                                                           |                                                                                                                                    |
+| **User-Friendly:** Write tests in plain English, making it accessible for non-programmers | **No Regular Updates:** Active development has ceased, which might affect future compatibility                                                                                                |
+| **Open-Source:** Free to use                                                              | **Low Community Support:** Very low community support as it is not used by many                                                                                                               |
+
+### When to use?
+
+- When your team includes non-technical members who need to write tests
+- For projects that would benefit from [behavior-driven development](https://cucumber.io/docs/bdd/) (BDD)
+
+## [**Appium**](https://appium.io/docs/en/latest/)
+
+Appium is a widely-used mobile test automation framework that supports UI automation of multiple platforms, including
+mobile (Android, iOS, Tizen), web, desktop, TV. both iOS and Android platforms. The popularity of this tool may stem
```

Comment:
The sentence `both iOS and Android platforms.` seems incomplete and thus breaks the flow of reading... 
maybe connect it with the previous sentence or make it a full sentence

Reply:
I forgot to remove that.